src/components/FocusableContent.tsx

```diff
@@ -35,7 +34,7 @@ interface FocusableContentProps {
 
 const FocusableContent: FC<FocusableContentProps> = ({ className, children, isFocused, onClose }) => {
   const { t } = useTranslation()
-  const modalRef = useFocusOnMount<HTMLDivElement>()
+  const modalRef = useRef<HTMLDivElement>(null)
```

Comment:
That's it? That's what causing the scroll?

Reply:
yep.